spec.html

```diff
@@ -23828,6 +24387,7 @@ <h1>HostImportModuleDynamically ( _referencingScriptOrModule_, _specifier_, _pro
               <dd>
                 <ul>
                   <li>At some future time, the host environment must perform FinishDynamicImport(_referencingScriptOrModule_, _specifier_, _promiseCapability_, an abrupt completion), with the abrupt completion representing the cause of failure.</li>
+                  <li>At some future time, the host environment must perform FinishDynamicImport(_referencingScriptOrModule_, _specifier_, _promiseCapability_, _promise_), where _promise_ is a Promise rejected with an error representing the cause of failure.</li>
```

Comment:
Both of these? Did you mean to replace the "abrupt completion" one?

Reply:
yeah looks like i had some odd merging. Thanks for catching that.